<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IDEWindow</class>
 <widget class="QMainWindow" name="IDEWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="console">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>691</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionOuvrir"/>
    <addaction name="actionEnregistrer"/>
    <addaction name="actionQuitter"/>
    <addaction name="actionImporter_Dossier"/>
   </widget>
   <widget class="QMenu" name="menuExecution">
    <property name="title">
     <string>Exécution</string>
    </property>
    <addaction name="actionExecuter"/>
   </widget>
   <widget class="QMenu" name="menuEditer">
    <property name="title">
     <string>Editer</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionCopier"/>
    <addaction name="actionCouper"/>
    <addaction name="actionColler"/>
   </widget>
   <widget class="QMenu" name="menuCompiler">
    <property name="title">
     <string>Compiler</string>
    </property>
    <addaction name="actionCompiler"/>
   </widget>
   <widget class="QMenu" name="menuAffichage">
    <property name="title">
     <string>Affichage</string>
    </property>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEditer"/>
   <addaction name="menuCompiler"/>
   <addaction name="menuExecution"/>
   <addaction name="menuAffichage"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOuvrir">
   <property name="text">
    <string>Ouvrir</string>
   </property>
  </action>
  <action name="actionEnregistrer">
   <property name="text">
    <string>Enregistrer</string>
   </property>
  </action>
  <action name="actionExecuter">
   <property name="text">
    <string>Exécuter</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionCompiler">
   <property name="text">
    <string>Compiler</string>
   </property>
  </action>
  <action name="toggleConsoleAction">
   <property name="text">
    <string>Afficher / Masquer console</string>
   </property>
  </action>
  <action name="actionImporter_Dossier">
   <property name="text">
    <string>Importer un dossier</string>
   </property>
  </action>
  <action name="actionCopier">
   <property name="text">
    <string>Copier</string>
   </property>
  </action>
  <action name="actionCouper">
   <property name="text">
    <string>Couper</string>
   </property>
  </action>
  <action name="actionColler">
   <property name="text">
    <string>Coller</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
